### PR TITLE
Meeting Minutes for End of November meeting.

### PR DESCRIPTION
**Pull request by SimonDarksideJ**
Minutes for the latest MonoGame Fuundation board meeting for publishing.

### CODE REVIEW

File: foundation/meeting-2023-11-22.md [hunk 1]
```diff
@@ -0,0 +1,54 @@
...
+- A schedule for contracts (or Issue Bounties) will be drawn up over the next few weeks/months to support necessary developments (although still the Foundation is offering their services freely to get criticval tasks moving), more on this space later.
+- An overall proposal format will be drafted for any bounties to be offered to paid work for the Foundation.  Principle rule is that no one "profits" from work, the funds are there to help support the work to be completed.
+- The RoadMap, work has begun in earnest and a formal roadmap will be published soon, but here are the highlights for the 3.X cycle:
+    - Native AOT support for cross platform builds.
+    - The addition of Vulkan support (which will also provide Metal support) for iOS/Android. (see [This issue](https://github.com/MonoGame/MonoGame/issues/4593) for details.)
+    - Update Xbox platform for DX 12 and GDKX support (with plans to bring DX12 to Windows once complete).
+    - Unity the Native API for MonoGame for all platforms, to make maintenance and new platforms easier.
```
[the remark this answers is not in the document]
Comment (SimonDarksideJ) [reply foundation/meeting-2023-11-22.md:42]: Freudian slip

**Review by SimonDarksideJ — COMMENTED** [review]
Comments addressed

File: foundation/meeting-2023-11-22.md [hunk 2]
```diff
@@ -0,0 +1,54 @@
...
+
+Here is a summary of the outcomes from our week long debate:
+
+- Funding will be international and the MonoGame Foundation has setup an acccount with Wise for international payments.  PayPal will be available as a last resort should developers have issues receiving funds.
+- A schedule for contracts (or Issue Bounties) will be drawn up over the next few weeks/months to support necessary developments (although still the Foundation is offering their services freely to get criticval tasks moving), more on this space later.
+- An overall proposal format will be drafted for any bounties to be offered to paid work for the Foundation.  Principle rule is that no one "profits" from work, the funds are there to help support the work to be completed.
+- The RoadMap, work has begun in earnest and a formal roadmap will be published soon, but here are the highlights for the 3.X cycle:
+    - Native AOT support for cross platform builds.
+    - The addition of Vulkan support (which will also provide Metal support) for iOS/Android. (see [This issue](https://github.com/MonoGame/MonoGame/issues/4593) for details.)
```
Comment (harry-cpp) [inline foundation/meeting-2023-11-22.md:40]: Its for both desktop and mobile platforms.

File: foundation/meeting-2023-11-22.md [hunk 3]
```diff
@@ -0,0 +1,54 @@
...
+- Marko Jeremic - Board Chairman
+- Thomas Altenburger - Board Member
+
+## Agenda
+- Funding discussions and the plans to offer funding support to community investments and work.
+- Initial discussion on which issues the MonoGame Foundation would like to fund first.
+- Technology highlights:
+    - Microsoft GDKX / GDK Support.
+    - Native AOT.
+    - Vulkan Support. (iOS/Android/MacOS?)
```
Comment (harry-cpp) [inline foundation/meeting-2023-11-22.md:21]: All desktop and mobile platforms.

File: foundation/meeting-2023-11-22.md [hunk 4]
```diff
@@ -0,0 +1,54 @@
...
+- The RoadMap, work has begun in earnest and a formal roadmap will be published soon, but here are the highlights for the 3.X cycle:
+    - Native AOT support for cross platform builds.
+    - The addition of Vulkan support (which will also provide Metal support) for iOS/Android. (see [This issue](https://github.com/MonoGame/MonoGame/issues/4593) for details.)
+    - Update Xbox platform for DX 12 and GDKX support (with plans to bring DX12 to Windows once complete).
+    - Unite the Native API for MonoGame for all platforms, to make maintenance and new platforms easier.
+- The content pipeline overhaul - Following increasing feedback regarding the Content Pipeline, the intention is to migrate the existing GUI tool to simply use a DotNet library instead, removing the need to maintain an external tool.  There will also be some internal updates to improve efficiency.  **Note* this will not change how the COntent Pipeline works in your projects, so there are no braking changes (although a Content Pipeline configuration will need to be migrated to the new pattern).
```
Comment (harry-cpp) [inline foundation/meeting-2023-11-22.md:43]: Well, there will still be a need for the content server-esk part of the tool, just saying "The content pipeline overhaul" is more then enough until the github issues are created with the final proposals.

File: foundation/meeting-2023-11-22.md [hunk 5]
```diff
@@ -0,0 +1,54 @@
...
+- Dependencies - The intention is to bring ALL dependencies under MonoGame control (either as direct forks or taking ownership), this will improve the overall reliability of components that MonoGame is heavily dependent on.
```
Comment (harry-cpp) [inline foundation/meeting-2023-11-22.md:44]: We are just trying to bring the building of them under our control, not the actual dev on them, so we can more easily update them, ensure they are statically linked, etc.

**Review by harry-cpp — CHANGES_REQUESTED** [review]

File: foundation/meeting-2023-11-22.md (hunk 3 above)
Comment (SimonDarksideJ) [reply foundation/meeting-2023-11-22.md:21]: Updated

File: foundation/meeting-2023-11-22.md [hunk 6]
```diff
@@ -0,0 +1,54 @@
...
+- Documentation overhaul - Updating, migrating and improving the MonoGame Documentation is a high priority for 3.x, currently under review by Simon Jackson and Dean Ellis.  SUpport wil lbe welcome by the community is centralising official documentation.
+- DotNet 8 upgrade plans - the next release of MonoGame (provisionally 3.9) will be upgraded to DotNet 8 to accelerate and enable other improvements to the MonoGame Framework.  This should cause **NO** breaking changes to existing projects, and if anything, offer a speed boots due to DotNet 8 compilation and runtime.  FOr those wanting to remain on DotNet 6, then you will have to remain on MG 3.8.
```
[the remark this answers is not in the document]
Comment (SimonDarksideJ) [reply foundation/meeting-2023-11-22.md:46]: Updated

File: foundation/meeting-2023-11-22.md (hunk 4 above)
Comment (SimonDarksideJ) [reply foundation/meeting-2023-11-22.md:43]: Simplified statement

**Review by dellis1972 — APPROVED** [review]

**Review by harry-cpp — CHANGES_REQUESTED** [review]

File: foundation/meeting-2023-11-22.md [hunk 7]
```diff
@@ -37,13 +37,13 @@ Here is a summary of the outcomes from our week long debate:
...
-- The content pipeline overhaul - Following increasing feedback regarding the Content Pipeline, the intention is to migrate the existing GUI tool to simply use a DotNet library instead, removing the need to maintain an external tool.  There will also be some internal updates to improve efficiency.  **Note* this will not change how the COntent Pipeline works in your projects, so there are no braking changes (although a Content Pipeline configuration will need to be migrated to the new pattern).
-- Dependencies - The intention is to bring ALL dependencies under MonoGame control (either as direct forks or taking ownership), this will improve the overall reliability of components that MonoGame is heavily dependent on.
+- The content pipeline overhaul - Following increasing feedback regarding the Content Pipeline, work has begun to impove the flow and management of content without affecting existing projects operation, details to follow.
+- Dependencies - The intention is to bring the building of ALL dependencies under MonoGame control (either as direct forks or taking ownership), this will improve the overall reliability of components that MonoGame is heavily dependent on.
```
[the remark this answers is not in the document]
Comment (SimonDarksideJ) [reply foundation/meeting-2023-11-22.md:44]: Removed clarification as it is not required

**Review by harry-cpp — APPROVED** [review]

**Pull request merged**